loadtest/user/userentity/actions.go

```diff
@@ -332,6 +332,21 @@ func (ue *UserEntity) GetTeam(teamId string) error {
 	return ue.store.SetTeam(team)
 }
 
+func (ue *UserEntity) GetTeamsForUser(userId string) ([]*model.Team, error) {
+	teams, resp := ue.client.GetTeamsForUser(userId, "")
```

Comment:
As previously discussed in #53 we should rename this to `GetTeams` and leverage the current user's id taken from the store.

Reply:
@streamer45 - Then that functionality is already implemented here https://github.com/mattermost/mattermost-load-test-ng/blob/03663efb16c1b1230f5a291117abbd8eb54f407f/loadtest/user/userentity/actions.go#L574. Should we close this issue then ?

Reply:
What I was suggesting was to make that function you highlighted just a wrapper of your `GetTeamsForUser` which while it won't see a lot of usage it could still come handy in the future as an *admin only* method.